### ISSUE TITLE
KCD Taiwan 2021

### ISSUE THREAD

**Issue by ricolin** [cncf/kubernetes-community-days#401]
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Location:
Online(Taiwan)
### Event Date and Time:
2021/12/03
### Event type: 
virtual
### Submission checklist:

*Please check each of these as you complete them:

* [*] No other Kubernetes Community Day event in the same region 
* [ ] Minimum of 3 organizers from 3 different organizations
* [*] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [*] Each organizer lives in or just outside of the KCD location
* [*] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer
* [*] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [ ] Minimum of 3 sponsors committed
* [*] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
* [*] Agree to offer (at minimum) 3 complimentary tickets to increase diversity for the event
* [*] Agree to use Cloud Native Community Groups for registration, scheduling, and event website
* [*] Agree to use SM Apply for CFP
* [*] Please list who should be added as an admin for the CFP:
* [*] Event date and time has been chosen
* [*] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [ ] Create a Twitter handle specific to your KCD event (except for countries like China and Korea where Twitter is not used) 
* [ ] Organizer email alias has been created
* [ ] Region and event has been created in Bevy

### Expected Number of Attendees:
300
### Fiscal sponsor that will accept payments
yes

**Comment by xmulligan** [comment]
@ricolin thanks for starting to pull this together! Let me know if you need any help or have questions along the way 😄

**Comment by xmulligan** [comment]
@ricolin I just wanted to check in on you and see if you needed anything from me. Have you found any sponsors yet?

**Comment by ricolin** [comment]
Hey @xmulligan We currently working on sponsorship doc and some related stuff, so no signed sponsor yet, but we do have some high hope from certain companies.

**Comment by xmulligan** [comment]
Great! Let me know if you need anything in the meantime. I'll be on vacation the 4-18 of July, but message Ihor in slack if you need anything

**Comment by xmulligan** [comment]
@ricolin how is your progress for sponsors?

**Comment by ricolin** [comment]
@xmulligan 
We already have three sponsor promised, but our sponsorship brochure is not released yet. We plan to do it next month.
On the other hand, can you help to generate KCD Taiwan logo? so we can put it in our brochure

**Comment by xmulligan** [comment]
@ricolin You can find the KCD logo here https://github.com/cncf/artwork/blob/master/examples/other.md#kubernetes-community-days. Is that ok or do you want something else?

**Comment by ricolin** [comment]
It's what I'm asking:) thanks

**Comment by xmulligan** [comment]
@ricolin who will be running the CfP? I can create the accounts now if you want

**Comment by ricolin** [comment]
@xmulligan That's great, Hazel Shen <hazel910159@gmail.com> will be moderating the CFP process (if that's what you asking). Please help to create the accounts for us.

**Comment by xmulligan** [comment]
@ricolin perfect, they should have an email with an invite to the platform now. Let me know if it didn't work.

Will you be creating an social media accounts for the event?

Also all organizers now need to take https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/. can you let me know when you have done so?

**Comment by xmulligan** [comment]
I would also like to schedule a monthly check in with you until the event so we have a regular check in point where I can help you with any issues that have arisen. Obviously, you can also contact me or have a meeting outside that time too. Can you suggest a time that works for you?

**Comment by xmulligan** [comment]
@ricolin to follow up on our emails. Can you update the date of the event above and schedule a monthly check in with me? calendly.com/billmulligan/1-1-with-bill-mulligan Let me know if you need a different time

**Comment by ricolin** [comment]
@xmulligan scheduled and change this to KCD Taiwan 2022 due to Pandemic reasons.

**Comment by xmulligan** [comment]
The planning for this event will begin again in February 2022 and the hope for a hybrid event in the summer

**Comment by ricolin** [comment]
@xmulligan As mentioned, the plan for 2022 is started, might require some discussion with organizers recently, will keep you updated

**Comment by xmulligan** [comment]
Thanks, let me know if you want to set up a time to chat. When do you think you will host the event?

**Comment by k8tgreenley** [comment]
@ricolin Are you not using the Bevy platfrom for registrations?

**Comment by ricolin** [comment]
@k8tgreenley No, as this time we're hosting physical event with COSCUP, we directly use their registration platform

**Comment by k8tgreenley** [comment]
@ricolin for my tracking purposes, how many registrations do you currently have? Do you have an agenda posted somewhere? How many CFP submissions did you receive?

**Comment by ricolin** [comment]
The registration is not yet open, so as agenda. As for CFP, it's about 14 received. We have already done with internal review, so I think agenda will be out soon. All agenda and event info can be found in https://coscup.org/2022/en/ We will update the official KCD website as well. will keep you informed

**Comment by k8tgreenley** [comment]
@ricolin thank you for letting me know. I have updated out notes to reflect. When you were approved for this event was it approved for you to use outside CFP for and registrations?

**Comment by ricolin** [comment]
@k8tgreenley yes we. We had a meeting with Bill and confirmed those details with him at the time.

**Comment by k8tgreenley** [comment]
Thank you @ricolin

**Comment by AudMonte01** [comment]
Hi @ricolin - a pleasure to meet you! I am the new KCD program manager and see your event is in about two weeks. How is everything going for KCD Taiwan? Do you have any questions or need some help? If not, @k8tgreenley and I would love to set up a post mortem with your team to hear how the event went. :-)

**Comment by pichuang** [comment]
Hi @AudMonte01 @k8tgreenley , We already have launch the event at here. The joint conference will be in-person event.

- [COSCUP ╳ KCD Taiwan 2022](https://community.cncf.io/events/details/cncf-kcd-taiwan-presents-coscup-x-kcd-taiwan-2022/)

Does anything should we need to know or do before we go to the event?

**Comment by AudMonte01** [comment]
Excellent! [Here is the timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=0) you might find useful, that you can make a copy of and use as a checklist. Let me know if you have any questions.

**Comment by pichuang** [comment]
Hi @AudMonte01 Is it possible to help us to fix the display issue on community chapter page?because there is no our community logo on the page.  https://community.cncf.io/chapters/

![telegram-cloud-photo-size-5-6255997858562027855-y](https://user-images.githubusercontent.com/5253671/180649102-6424a707-706c-4937-ae27-8efa259ac29d.jpg)

![telegram-cloud-photo-size-5-6255997858562027856-y](https://user-images.githubusercontent.com/5253671/180649115-5848eab9-f299-44a6-9c8c-843a5645d30d.jpg)

**Comment by AudMonte01** [comment]
Hi @pichuang - It looks like the region in your chapter was missing, so I added it for you. Please see below that Taiwan is now showing under Community Groups->Asia
<img width="1129" alt="Screen Shot 2022-07-25 at 11 56 20 AM" src="https://user-images.githubusercontent.com/107572090/180853499-b51a6d29-b703-4950-a951-3178f2c0da95.png">

**Comment by pichuang** [comment]
Hi @AudMonte01 One more typo. Could you please modify `Location: Taipeh, Taiwan` to `Location: Taipei, Taiwan`

<img width="1704" alt="Screen Shot 2022-07-26 at 10 33 26" src="https://user-images.githubusercontent.com/5253671/180911125-a70aec10-d62b-4b90-a958-bd7d4f76c545.png">

Thanks for your support

**Comment by AudMonte01** [comment]
No problem - this has been fixed @pichuang .

**Comment by AudMonte01** [comment]
Hi @pichuang - I would like to set up a call with you and team to hear about how KCD Taiwan went, as well as gather analytics for your event. Here is my Calendly to choose a time that works for you: https://calendly.com/audmonte/30min

**Comment by pichuang** [comment]
Number of Day 1 Attendees is 1,561
Number of Day 2 Attendees is 1,321

**Comment by AudMonte01** [comment]
Thank you, Phil and team for these numbers. I look forward to learning more
in our meeting.

Today is a US holiday, so I moved our meeting to tomorrow at the same time.
I hope that will work for you.  Please let me know if it does not, or you
need an earlier time (I can meet one hour earlier if needed).

Thanks again,
Audra

**Comment by pichuang** [comment]
Hi @AudMonte01 I won't be able to attend the meeting today as something has come up. Could we put off the meeting until this Thu. or Fri.?

**Comment by AudMonte01** [comment]
That is not a problem. Can we meet on Thursday at 7:00pm IST which is
6:30am my time (Pacific Standard)?

**Comment by pichuang** [comment]
Hi @AudMonte01 , the schedule is available for me. Do we need to prepare any information before attending the meeting?

**Comment by AudMonte01** [comment]
Wonderful! The following would be helpful to know:

   - How many registered for your event
   - What are the total number of unique attendees
   - Confirm # of speakers (I see 10)
   - Confirm registration goal (I see 300)
   - Ticket cost (if any)
   - Registration platform (if different from Bevy)
   - Let us know about your experience and what we (the CNCF) could do
   better next time

Thank you!

**Comment by pichuang** [comment]
- How many registered for your event

I need to double-check with the general coordinator. will give you feedback soon.

- What are the total number of unique attendees

Due to KCD Taiwan co-event with COSCUP on the 1st day, the number of attendees is 1,561.

- Confirm # of speakers (I see 10)

Actually, 12 speakers because the 2 speakers are from COSCUP Main track instead of KCD Taiwan.

- Confirm registration goal (I see 300)

Yes. Over 300. so we over archived the target.

- Ticket cost (if any)

Free for attendees. COSCUP x KCD Taiwan had many [sponsors][1] in 2022. So we had all free tickets for attendees.

- Registration platform (if different from Bevy)

[Opass][2], that is FLOSS conference check-in solution, mostly used in Taiwan.

- Let us know about your experience and what we (the CNCF) could do better next time

For the event, our target markets is Taiwan, China, HK, and Southeast Asia.
I propose CNCF may bridge and lead the regional-level resources with each KCD event. for example, speakers who want to oversea share and sponsor relationships.

[1]: https://coscup.org/2022/en/sponsor
[2]: https://opass.app/

**Comment by AudMonte01** [comment]
Thank you for the information ahead of the meeting.

In the meeting on Thursday, we can discuss

   - the number of registrations
   - how you liked using Opass <https://opass.app/> and if you
   recommend other regions using it
   - elaborate on how the CNCF may bridge and lead the regional-level
   resources with each KCD event. Thank you for your example - I will ask to
   elaborate on what "speakers who want to oversea share" means


Kind regards,

**Comment by AudMonte01** [comment]
Thank you @pichuang and team for meeting me today. Here are the agenda items we have to follow-up on:
* Registrations for KCD specific event within COSCUP to be sent to Audra the week of 9/12
* How many sponsors for just the KCD event (can not see in COSCUP survey)
* Videos - The YouTube link to be shared with Audra
* Audra to create CNCF Taiwan Chapter and connect to community team, as well as look into a CNCF logo that can be used

**Comment by AudMonte01** [comment]
Hi @pichuang - I have also added you four organizers to the Cloud Native Taiwan Chapter in Bevy as members. Please let me know if you have received an email notification for this.

Here is where you can create your meetups and see others happening in the region.

**Comment by AudMonte01** [comment]
Hello @pichuang - were you able to find out how many people registered for this KCD specifically (not the COSCUP event in as a whole)? Also, is the 1,561 number attendance for the COSCUP event total, and not just your KCD?

Thank you kindly

**Comment by pichuang** [comment]
Hi @AudMonte01, sorry for replying late. The 1,561 number attendance is for the COSCUP event. Based on the event recording, the maximum attendance for KCD Taiwan is 131, and the avg. is around 100.

**Comment by AudMonte01** [comment]
Thank you @pichuang - that makes sense for an in-person KCD, as it is similar to others that are part of co-located events given the rooms given can manage up to 300 at most.

Do you have any idea of how many people signed up for the KCD (usually a higher number than the attendance by 50-70% more).

**Comment by pichuang** [comment]
Hi @AudMonte01, Regarding the event is free for everyone, unfortunately, we don't have the specific number for signing up.

**Comment by AudMonte01** [comment]
Okay - thank you @pichuang for confirming there was no registration required for this KCD
